From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Oleg <balbes-150@yandex.ru>
Date: Mon, 4 Jan 2021 11:51:43 +0100
Subject: [ARCHEOLOGY] add station legacy (#2521)

> X-Git-Archeology: - Revision a1e96e68d864ddc2fef169f3f503a9493311313b: https://github.com/armbian/build/commit/a1e96e68d864ddc2fef169f3f503a9493311313b
> X-Git-Archeology:   Date: Mon, 04 Jan 2021 11:51:43 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: add station legacy (#2521)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0cdffb29b07305209efb12cf3b5ac6032d3a1153: https://github.com/armbian/build/commit/0cdffb29b07305209efb12cf3b5ac6032d3a1153
> X-Git-Archeology:   Date: Wed, 24 Mar 2021 19:01:53 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Renaming DEV branch to EDGE (#2704)
> X-Git-Archeology:
---
 arch/arm64/boot/dts/rockchip/rk3328-linux.dtsi | 41 ++++++++++
 1 file changed, 41 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328-linux.dtsi b/arch/arm64/boot/dts/rockchip/rk3328-linux.dtsi
new file mode 100644
index 000000000000..77e9dbf874eb
--- /dev/null
+++ b/arch/arm64/boot/dts/rockchip/rk3328-linux.dtsi
@@ -0,0 +1,41 @@
+/ {
+
+	reserved-memory {
+		#address-cells = <2>;
+		#size-cells = <2>;
+		ranges;
+
+		drm_logo: drm-logo@00000000 {
+			compatible = "rockchip,drm-logo";
+			reg = <0x0 0x0 0x0 0x0>;
+		};
+	};
+};
+
+&display_subsystem {
+	logo-memory-region = <&drm_logo>;
+	status = "okay";
+	route {
+		route_hdmi: route-hdmi {
+			status = "disabled";
+			logo,uboot = "logo.bmp";
+			logo,kernel = "logo_kernel.bmp";
+			logo,mode = "center";
+			charge_logo,mode = "center";
+			connect = <&vop_out_hdmi>;
+		};
+
+		route_tve: route-tve {
+			status = "okay";
+			logo,uboot = "logo.bmp";
+			logo,kernel = "logo_kernel.bmp";
+			logo,mode = "fullscreen";
+			charge_logo,mode = "fullscreen";
+			connect = <&vop_out_tve>;
+		};
+	};
+};
+
+&rga {
+	status = "okay";
+};
-- 
Armbian

